server/app/services/applicant/question/ApplicantQuestion.java

```diff
@@ -133,6 +134,12 @@ public String getQuestionText() {
     return repeatedEntity.map(r -> r.contextualize(text)).orElse(text);
   }
 
+  /** Sanitized HTML for the question text that processes Markdown. */
+  public String getFormattedQuestionText() {
+    return TextFormatter.formatTextToSanitizedHTML(
+        getQuestionText(), /* preserveEmptyLines= */ true, /* addRequiredIndicator= */ false);
```

Comment:
suggestion (maybe blocking): I'm a little unclear about which part of the old code this has parity with, but if it controls how applicants see questions then I'm pretty sure we need to pass in the logic for required indicator like we do here: https://github.com/civiform/civiform/blob/0f1451b6127294d7a79f3c18ae639d951f139da7/server/app/views/questiontypes/ApplicantQuestionRendererImpl.java#L106

Reason being that we place the required indicator in different places depending on if the question text ends in a list: https://github.com/civiform/civiform/blob/0f1451b6127294d7a79f3c18ae639d951f139da7/server/app/views/components/TextFormatter.java#L108

Reply:
done. I updated to pass in whether to add a required indicator (also including not passing it in for address/date/name questions because the * appears on the individual inputs, or at least will once my next CSS pass is done)

Reply:
I only checked name, but I think that's true for address and date too.

Reply:
the mocks had shown the asterisk missing for those question types, but I chatted with Sijia and we decided to add them back. Updated to restore it.